Description: Set default printing options for lp
 Change default printing options, switch from lpr to lp.
Author: Julien BLACHE <jblache@debian.org>
Forwarded: not-needed

Index: xsane-0.998/src/xsane.h
===================================================================
--- xsane-0.998.orig/src/xsane.h	2010-11-16 22:00:33.000000000 +0100
+++ xsane-0.998/src/xsane.h	2011-02-04 19:50:22.945016002 +0100
@@ -510,8 +510,8 @@
 
 #define OUT_FILENAME     	"out.pnm"
 #define PRINTERNAME	  	"new printer"
-#define PRINTERCOMMAND  	"lpr"
-#define PRINTERCOPYNUMBEROPTION "-#"
+#define PRINTERCOMMAND  	"lp"
+#define PRINTERCOPYNUMBEROPTION "-n"
 #define FAXPROJECT 	    	"faxproject"
 #define FAXCOMMAND 	 	"sendfax"
 #define FAXRECEIVEROPT		"-d"
